CHANGELOG.md

```diff
@@ -5,6 +5,104 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [0.46.0] - 2025-04-22
+#### ✨ Highlights
+Breaking change for the new `args` feature we introduced in the last release.
+As the `args` names in the `tasks` are now not allowed to have dashes(`-`), because of the introduction of `Minijinja` for rendering the tasks and possibly reading them as a minus.
+
+This release comes with another set of features for the `tasks`!
+-  The command of a task is now able to use `minijinja` for rendering the command. Including the `args` as variables in the template.
+```toml
+ [tasks]
+# The arg `text`, converted to uppercase, will be printed.
+task1 = { cmd = "echo {{ text | upper }}", args = ["text"] }
+# If arg `text` contains 'hoi', it will be converted to lowercase. The result will be printed.
+task2 = { cmd = "echo {{ text | lower if 'hoi' in text }}", args = [
+  { arg = "text", default = "" },
+] }
+# With `a` and `b` being strings, they will be appended and then printed.
+task3 = { cmd = "echo {{ a + b }}", args = ["a", { arg = "b", default = "!" }] }
+# `names` will be split by whitespace and then every name will be printed separately
+task4 = { cmd = "{% for name in names | split %} echo {{ name }};{% endfor %}", args = [
+  "names",
+] }
+```
+-  The `depends-on` key can now include the environment that the task should run in.
+```toml
+[tasks.test-all]
+depends-on = [
+  { task = "test", environment = "py311" },
+  { task = "test", environment = "py312" },
+]
+```
```

Comment:
```suggestion
```toml
[tasks]
test-all = { depends-on = [
  { task = "test", environment = "py311" },
  { task = "test", environment = "py312" },
] }
```

We also introduced a shorter version of tasks that only contain task dependencies.
Therefore, the example above can also be written like this:

```toml
[tasks]
test-all = [
  { task = "test", environment = "py311" },
  { task = "test", environment = "py312" },
]
```

Reply:
Good one changed it!